lega/conf/defaults.ini

```
@@ -45,6 +45,17 @@ connection = amqps://guest:guest@mq:5672/%2F?heartbeat=0&connection_attempts=30&
 verify_peer = true
 verify_hostname = no
 
+# ssl = true
+# username = guest
+# password = guest
+# hostname = mq
+# port = 5672
+# virtual_host = /
+# heartbeat = 0
+# connection_attempts = 30
+# retry_delay = 10
+
```

```suggestion
ssl = true
username = guest
password = guest
hostname = mq
port = 5672
virtual_host = /
heartbeat = 0
connection_attempts = 30
retry_delay = 10

```
These should probably not be commented out.